### PR TITLE
Support for new resource syntax in PBSPro

### PR DESCRIPTION
**Pull request by Phhere**
PBSPro has deprecated the support for nodes=..:ppn=... since some versions and you should use the new syntax for it.

### CODE REVIEW

**Comment by Phhere** [comment]
I have found some more issues with PBSPro and will fix them

**Comment by adamnovak** [comment]
@Phhere Thanks for working on this. Is it still WIP, or is it ready for me to pull into the Toil repo for integration testing?

**Comment by Phhere** [comment]
I am currently testing it togeather with cactus. when the pipeline is done, it can be merged i think

**Comment by mr-c** [comment]
Any update @phhere? Thanks again for your contribution!

**Comment by adamnovak** [comment]
I've set this up for testing. If it passes and you think it works @mr-c, I suppose we can just pull it in.

File: src/toil/batchSystems/torque.py [hunk 1]
```diff
@@ -163,12 +163,21 @@ def prepareQsub(self, cpu, mem, jobID):
...
-                memStr = str(old_div(mem, 1024)) + 'K'
-                reqline.append('mem=' + memStr)
+            if self._version == "pro":
+                request = 'select=1'
+                if mem is not None:
+                    memStr = str(old_div(mem, 1024)) + 'K'
+                    request +=':mem=' + memStr
+                if cpu is not None and math.ceil(cpu) > 1:
+                    request +=':ncpus=' + str(int(math.ceil(cpu)))
+                 reqline.append(request)
```
Comment (mr-c) [inline src/toil/batchSystems/torque.py:173]: error: unindent does not match any outer indentation level

**Comment by mr-c** [comment]
I pushed this branch to our repo as `issues/3863-newer-pbspro` so that GitLabCI would run

**Pull request merged**